### PR TITLE
Side 2173 make all uint8array take string

### PR DESCRIPTION
**Pull request by RusGraf**

### CODE REVIEW

**Review by MaggieMaSS — APPROVED** [review]

File: ldk/javascript/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   ],
   "scripts": {
     "build": "rm -rf dist && tsc",
+    "prepush": "npm run build && npm run test && npm run lint",
```
Comment (cshields-oliveai) [inline ldk/javascript/package.json:21]: *high five*

Comment (RusGraf) [reply ldk/javascript/package.json:21]: hehe, 1 step at a time.

File: ldk/javascript/src/utils/network.mapper.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import { TextDecoder } from 'text-encoding-shim';
```
Comment (cshields-oliveai) [inline ldk/javascript/src/utils/network.mapper.ts:1]: Would it be worthwhile to move this to `src/network/network.mapper.ts`? It's becoming apparent that we will need mappers for many aptitudes, and I thought putting individual mappers closer to them might make sense.

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/src/utils/network.mapper.ts (hunk 2 above)
Comment (RusGraf) [reply ldk/javascript/src/utils/network.mapper.ts:1]: I'm actually not sure what would be the most idiomatic way here. I saw applications with both structures. Do you know specifically why would you do one way vs another?

File: ldk/javascript/package.json (hunk 1 above)
Comment (cshields-oliveai) [reply ldk/javascript/package.json:21]: A thought also occurred to me - what about including building the `examples` dir as well to help prevent those projects from going stale?

Comment (RusGraf) [reply ldk/javascript/package.json:21]: Hmm, I think it is a good idea to run them, but I'm wondering where would be best to do it. Perhaps it is better fits to do it in github actions via hooks?

Comment (cshields-oliveai) [reply ldk/javascript/package.json:21]: Maybe we can create a new user story for this.

**Pull request merged**